ckanext/unaids/plugin.py

```diff
@@ -102,10 +100,10 @@ def get_helpers(self):
             u'get_autogenerated_resources': get_autogenerated_resources,
             u'get_user_obj': get_user_obj,
             u'get_all_organizations': get_all_organizations,
-            u'extstorage_resource_authz_scope': get_extstorage_resource_authz_scope,
-            u'extstorage_resource_storage_prefix': get_extstorage_resource_storage_prefix,
-            u'extstorage_resource_filename': get_extstorage_resource_filename,
-            u'max_resource_size': get_max_resource_size
+            u'extstorage_resource_authz_scope': extstorage_helpers.resource_authz_scope,
+            u'extstorage_resource_storage_prefix': extstorage_helpers.resource_storage_prefix,
+            u'extstorage_resource_filename': extstorage_helpers.resource_filename,
+            u'max_resource_size': uploader.get_max_resource_size()
```

Comment:
I would keep the value as function not a value. This is very confusing. You'd expect to get it called like:
```
helpers.max_resource_size()
```